modules/arm_plugin/src/arm_converter/arm_converter.hpp

```diff
@@ -537,8 +541,8 @@ template<typename ...T0, typename ...T1>
 constexpr static std::tuple<T0..., T1...> merge(std::tuple<T0...>, std::tuple<T1...>) {return {};}
 
 constexpr static auto boolType = std::tuple<bool>{};
-constexpr static auto intTypes = std::tuple<std::int8_t, std::uint8_t, std::int16_t, std::uint16_t, std::int32_t, std::uint32_t, std::int64_t>{};
+constexpr static auto intTypes = std::tuple<std::int8_t, std::uint8_t, std::int16_t, std::uint16_t, std::int32_t, std::uint32_t, std::int64_t, std::uint64_t>{};
 constexpr static auto indexTypes = std::tuple<std::int32_t, std::int64_t>{};
-constexpr static auto floatTypes = std::tuple<ngraph::float16, float, double>{};
+constexpr static auto floatTypes = std::tuple<ngraph::bfloat16, ngraph::float16, float, double>{};
```

Comment:
maybe it's better to add bfloat16 to concat / reshape types specifically? Otherwise, a lot of reference implementations will be instantiated for float types, but never used